src/lower/mode_format_singleton.cpp

```diff
@@ -10,12 +10,12 @@ using namespace taco::ir;
 namespace taco {
 
 SingletonModeFormat::SingletonModeFormat() : 
-    SingletonModeFormat(false, true, true) {
+    SingletonModeFormat(false, true, true, false) {
 }
 
 SingletonModeFormat::SingletonModeFormat(bool isFull, bool isOrdered,
-                                         bool isUnique, long long allocSize) :
```

Comment:
Oh, you are right. Thank you. By the way, it is wired because I modified the code as you suggested about a week ago, but the pull request does not seem to reflect it.